contracts/eth/nearbridge/contracts/NearDecoder.sol

```diff
@@ -7,6 +7,9 @@ library NearDecoder {
     using Borsh for Borsh.Data;
     using NearDecoder for Borsh.Data;
 
+    uint8 constant VALIDATOR_V1 = 0;
+    uint8 constant VALIDATOR_V2 = 1;
```

Comment:
FYI: When this issue is solved (https://github.com/near/nearcore/issues/5471) we should be using the PROTOCOL_VERSION (from the block) rather than this hardcoded version. Not action item here, just a comment.

Reply:
Yes, I saw that. But I don't expect this will happen soon.